src/translations/pages/assistant.ts

```diff
@@ -2,6 +2,11 @@ import { translation as _ } from '@atb/translations/commons';
 
 export const Assistant = {
   title: _('Planlegg reisen', 'Plan travel', 'Planlegg reisa'),
+  homeLink: _(
+    'Tilbake til forsiden',
+    'Back to the front page',
+    'Tilbake til framsida',
```

Comment:
Would it be better to have `Tilbake til frammr.no` here?

Reply:
I belive "Tilbake til frammr.no" would be a slightly clearer explanation.